src/components/useraccount/TabReturn/TabReturn.js

```diff
@@ -43,7 +48,8 @@ export default {
             items: this.selectedItems,
           },
         }).then(() => {
-          this.$router.push({ name: 'orders' })
```

Comment:
@plaskevich  You can use [this abstraction](https://github.com/commercetools/sunrise-spa/blob/master/src/components/common/shared.js#L82-L98) instead.

Reply:
I changed the code